### ISSUE TITLE
Assessing GHA workflows more precisely?

### ISSUE THREAD

**Issue by maelle** [ropensci-review-tools/pkgcheck#136]
E.g. could pkgcheck detect if v2 of r-lib/actions are used?

**Comment by mpadge** [comment]
Yes, easily. That's likely a good check to implement here - shall i have a go in the next coupla weeks, or do you want to try that? Or @assignUser if you'd be interested in attempting this? Your insight into our general way of inserting new checks would likely be really useful so please let us know if you'd be interested?

**Comment by assignUser** [comment]
So:
- detect use of r-lib/actions
- check for `v2`
-  raise note in CI section or error?

**Comment by maelle** [comment]
I'd say error?

**Comment by mpadge** [comment]
Slightly more clearly:

- detect presence of a `pkgcheck` file in `.github/workflows`
- check for `v2` of the main actions
- Follow procedure described [in vignette](https://docs.ropensci.org/pkgcheck/articles/extending-checks.html) to write a check function, and to implement an `output` function with a condition `!checks$checks$uses_actions_v2`. The package should take care of the rest.

But only if you (@assignUser)  want to do this - no worries if not i'll be happy to implement. Just let us know please.

**Comment by assignUser** [comment]
I am interested in doing this but I won't have time this week, if that's ok?

**Comment by mpadge** [comment]
No worries - there's no rush at all. Whenever you can find time will be great. Thanks!

**Comment by maelle** [comment]
> detect presence of a pkgcheck file in .github/workflows

What do you mean by that file?

**Comment by maelle** [comment]
Oh, I was confused, I misread your earlier comment, thank you.
